metrics/metrics.go

```diff
@@ -6,13 +6,27 @@ package metrics
 
 import (
 	"context"
+	"os"
+	"strconv"
 	"time"
 
 	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/drone/autoscaler"
 )
 
 var noContext = context.Background()
 
+// this is a feature flag that can be used to enable
+// metrics to track registering/unregistering of servers
+var registerKnownServers = false
+
+func init() {
+	registerKnownServers, _ = strconv.ParseBool(
+		os.Getenv("DRONE_AUTOSCALER_REGISTER_KNOWN_SERVERS"),
```

Comment:
can you add this to https://github.com/drone/autoscaler/blob/master/config/config.go & inject the config struct, similar to how we do it elsewhere please

Reply:
okay, pushed, can you re-review please?